api/spec/models/log_entry_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe LogEntry, type: :model do
+  let(:log_entry) { described_class.new(user_id: 1, event_id: 1, action: :event_member_created) }
+
+  describe "validations" do
+    subject { log_entry }
+
+    it { should validate_presence_of(:user_id) }
+    it { should validate_presence_of(:event_id) }
+    it { should validate_presence_of(:action) }
+  end
+
+  describe "actions enum (MongoidEnum)" do
```

Comment:
You're not testing in this spec that valid values pass. In general, it would be good to have good coverage for the actions enum, including the methods your class adds dynamically for the enum.